collection-scripts/gather-istio

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
re-name this file `gather_istio` to stay consistent with other commands in the collection scripts repo.